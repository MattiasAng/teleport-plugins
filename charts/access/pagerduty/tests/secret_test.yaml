suite: Test secret
templates:
  - secret.yaml
tests:
  - it: should contain the api key
    set:
      pagerduty:
        apiKey: mymattermostapikey
    asserts:
      - matchSnapshot: {}

  - it: should not exist when using external secret
    set:
      pagerduty:
        apiKeyFromSecret: my-pagerduty-secret
    asserts:
      - hasDocuments:
          count: 0
