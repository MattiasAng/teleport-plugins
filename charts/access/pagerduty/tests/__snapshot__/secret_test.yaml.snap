should contain the api key:
  1: |
    apiVersion: v1
    data:
      pagerdutyApiKey: bXltYXR0ZXJtb3N0YXBpa2V5
    kind: Secret
    metadata:
      name: RELEASE-NAME-teleport-plugin-pagerduty-secret
    type: Opaque
